ballerina-tests/tests/07_unary_server_pb.bal

```diff
@@ -193,7 +193,7 @@ public isolated client class HelloWorld100Client {
         } else {
             message = req;
         }
-        var payload = check self.grpcClient->executeSimpleRPC("grpcservices.HelloWorld100/testNoResponse", message, headers);
+        var _ = check self.grpcClient->executeSimpleRPC("grpcservices.HelloWorld100/testNoResponse", message, headers);
```

Comment:
Better if we can remove this `var` when we skip the variable since it is unnecessary.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/565/commits/75b3e207b156fa27d761f578923cd97f8e4020a2